tools/maint/update_settings_docs.py

```diff
@@ -101,7 +101,7 @@ def write_file(f):
       # Format:
       #   var NAME = DEFAULT;
       # Split it and strip the final ';'.
-      _, setting_name, _, setting_default = line.strip(';').split()
+      _, setting_name, _, setting_default = line.strip(';').split(None, 3)
```

Comment:
What is this for?

Reply:
This code parses the settings.js file and emits the `.rst` docs.

Without this change it cannot parse a list on single line because it has extra whitespace chars.  Adding 3 there means we never get more than 4 results from the split, which is what we want.